src/uikit.cs

```diff
@@ -4838,6 +4842,12 @@ interface UIColor : NSSecureCoding, NSCopying {
 		[Export ("getRed:green:blue:alpha:")]
 		bool GetRGBA2 (out nfloat red, out nfloat green, out nfloat blue, out nfloat alpha);
 #endif
+
+		// From the NSItemProviderWriting protocol, a static method.
+		[NoWatch, NoTV, Mac (10,13), iOS (11,0)]
+		[Static]
+		[Export ("writableTypeIdentifiersForItemProvider", ArgumentSemantic.Copy)]
+		string[] WritableTypeIdentifiers { get; }
```

Comment:
UIColor and UIImage don't appear to implement the other static method.  Attempting to use it in Xcode gives me:

```'NSInvalidArgumentException', reason: '+[UIColor itemProviderVisibilityForRepresentationWithTypeIdentifier:]: unrecognized selector sent to class 0x10f2719b0'```